tests/functional/test_reshaping.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
+import numpy as np
+
+from utils.grpc import infer_batch
+from utils.rest import infer_batch_rest
+from constants import PREDICTION_SERVICE, ERROR_SHAPE
+
+
+shapes = [
+    {'in': (1, 3, 300, 300), 'out': (1, 1, 200, 7)},
+    {'in': (1, 3, 500, 500), 'out': (1, 1, 200, 7)},
+    {'in': (1, 3, 224, 224), 'out': (1, 1, 200, 7)},
+    {'in': (4, 3, 224, 224), 'out': (1, 1, 800, 7)},
+    {'in': (8, 3, 312, 142), 'out': (1, 1, 1600, 7)},
+    {'in': (1, 3, 1024, 1024), 'out': (1, 1, 200, 7)},
+]
+
+
+class TestModelReshaping:
+
+    def test_single_local_model_reshaping(
+            self, face_detection_model_downloader,
+            start_server_face_detection_model,
+            create_grpc_channel):
+
+        print("Downloaded model files:", face_detection_model_downloader)
```

Comment:
Overall this fixture is required for `start_server_face_detection_model`, so I don't know if it should be here, because this test dependency chart is:
face_detection_model_downloader -> start_server_face_detection_model -> test
So if you already have it as dependency in `start_server_face_detection_model`, it unnecessary here.

If it helps in writing tests, I am ok to leave that line :)

Reply:
If you're okay with it, I would keep that line to be consistent with other tests. It's session scoped fixture so it gets called one time anyway.